src/commands/index-command.php

```diff
@@ -170,9 +170,9 @@ protected function run_indexation_actions( $assoc_args ) {
 			$this->clear();
 
 			// Delete the transients to make sure re-indexing runs every time.
-			\delete_transient( Indexable_Post_Indexation_Action::TRANSIENT_CACHE_KEY );
-			\delete_transient( Indexable_Post_Type_Archive_Indexation_Action::TRANSIENT_CACHE_KEY );
-			\delete_transient( Indexable_Term_Indexation_Action::TRANSIENT_CACHE_KEY );
+			\delete_transient( Indexable_Post_Indexation_Action::UNINDEXED_COUNT_TRANSIENT );
+			\delete_transient( Indexable_Post_Type_Archive_Indexation_Action::UNINDEXED_COUNT_TRANSIENT );
+			\delete_transient( Indexable_Term_Indexation_Action::UNINDEXED_COUNT_TRANSIENT );
```

Comment:
Do we need to provide an upgrade routine now that the transients have been renamed? 🤔 

A routine that moves any pre-existing old transients to the ones with the new values.

Or would that be overkill?

Reply:
I don't think that is necessary for a transient that is only around for a day.